.github/dependabot.yml

```diff
@@ -4,33 +4,33 @@ updates:
   directory: "/"
   schedule:
     interval: daily
-    time: "00:00"
+    time: "22:00"
     timezone: "America/Los_Angeles"
   open-pull-requests-limit: 25
```

Comment:
In a separate PR, Jason suggested reducing this to like 10 or something, maybe we can pull that forward?

Reply:
I'll put that in a follow-up PR. CI/CD have to run either way.